tests/validation/tests/v3_api/test_monitoring.py

```diff
@@ -0,0 +1,299 @@
+import pytest
+import copy
+from .common import *  # NOQA
+
+namespace = {
+    "cluster": None,
+    "project": None,
+}
+
+cluster_query_template = {
+    "obj": None,
+    "action_name": "query",
+    "filters": {},
+    "metricParams": {},
+    "interval": "5s",
+    "isDetails": True,
+    "from": "now-5s",
+    "to": "now"
+}
+
+cluster_graph_list = [
+    "cluster-network-packet",
+    "cluster-network-io",
+    "cluster-disk-io",
+    "cluster-cpu-load",
+    "cluster-cpu-usage",
+    "cluster-fs-usage-percent",
+    "cluster-memory-usage",
+]
+
+etcd_graph_list = [
+    "etcd-grpc-client",
+    "etcd-stream",
+    "etcd-raft-proposals",
+    "etcd-server-leader-sum",
+    "etcd-db-bytes-sum",
+    "etcd-sync-duration",
+    "etcd-server-failed-proposal",
+    "etcd-leader-change",
+    "etcd-rpc-rate",
+]
+
+kube_component_graph_list = [
+    "scheduler-total-preemption-attempts",
+    "ingresscontroller-nginx-connection",
+    "apiserver-request-count",
+    "controllermanager-queue-depth",
+    "scheduler-e-2-e-scheduling-latency-seconds-quantile",
+    "scheduler-pod-unscheduler",
+    "apiserver-request-latency",
+]
+
+node_graph_list = [
+    "node-network-packet",
+    "node-network-io",
+    "node-fs-usage-percent",
+    "node-cpu-load",
+    "node-disk-io",
+    "node-memory-usage",
+    "node-cpu-usage",
+]
+
+rancher_component_graph_list = [
+    "fluentd-buffer-queue-length",
+    "fluentd-input-record-number",
+    "fluentd-output-errors",
+    "fluentd-output-record-number",
+]
+
+workload_graph_list = [
+    "workload-network-packet",
+    "workload-memory-usage-bytes-sum",
+    "workload-cpu-usage",
+    "workload-network-io",
+    "workload-disk-io",
+]
+
+name_mapping = {
+    "cluster": cluster_graph_list,
+    "etcd": etcd_graph_list,
+    "kube-component": kube_component_graph_list,
+    "rancher-component": rancher_component_graph_list,
+    "workload": workload_graph_list,
+    "node": node_graph_list,
+}
+
+CLUSTER_MONITORING_APP = "cluster-monitoring"
+MONITORING_OPERATOR_APP = "monitoring-operator"
+PROJECT_MONITORING_APP = "project-monitoring"
+
+
+def test_monitoring_enable_cluster_monitoring():
```

Comment:
@jiaqiluo Instead of having enabling cluster monitoring as a test case , we should be able to move it as a pre requirement for all the existing test cases and move it a module scoped fixture with "autoUse" set to "True"

Reply:
fixed